tests/behat/features/filter-campaigns.feature

```diff
@@ -0,0 +1,69 @@
+@javascript @retry
+Feature: Manage campaigns
+  As a cms author
+  I want to search/filter campaigns within the CMS
+  So that I can find specific campaigns quickly
+
+  Background:
+    Given a "ChangeSet" "First Campaign" with "Description"="this is a campaign"
+      And a "ChangeSet" "Second Campaign" with "Description"="lorem ipsum" and "State"="published" and "PublishDate"="2025-01-01 12:00:00"
+      And a "ChangeSet" "Third Campaign" with "Description"="lorem ipsum" and "State"="published" and "PublishDate"="2025-05-06 12:00:00"
+      And the "group" "CAMPAIGNS_EDITOR" has permissions "Access to 'Campaigns' section"
+      And I am logged in as a member of "CAMPAIGNS_EDITOR" group
+      And I go to "/admin/campaigns"
```

Comment:
```suggestion
    Given a "ChangeSet" "First Campaign" with "Description"="this is a campaign"
    And a "ChangeSet" "Second Campaign" with "Description"="lorem ipsum" and "State"="published" and "PublishDate"="2025-01-01 12:00:00"
    And a "ChangeSet" "Third Campaign" with "Description"="lorem ipsum" and "State"="published" and "PublishDate"="2025-05-06 12:00:00"
    And the "group" "CAMPAIGNS_EDITOR" has permissions "Access to 'Campaigns' section"
    And I am logged in as a member of "CAMPAIGNS_EDITOR" group
    And I go to "/admin/campaigns"
```

And unident all the other bits in this file

Reply:
Indentation is all over the place in feature files. Unless we're going to adopt a specific style guide for them I don't think it's worth either of our time worrying about it.

Even within this module (and even within some individual feature files) there's already a mix of indented and not-indented `And` lines (though it looks like the slight majority in this module _do_ indent them).